packages/app/src/elements/login-signup.ts

```diff
@@ -719,12 +720,14 @@ export class LoginOrSignup extends StartForm {
                             <div class="spacer"></div>
 
                             <div class="hint">
-                                Hi there, <strong>${this._nameInput?.value || "Stranger"}</strong>! Let's set up your
-                                brand new ${process.env.PL_APP_NAME} account! (This will only take a few moments.)
+                                Hi there, <strong class="break-words">${this._nameInput?.value || "Stranger"}</strong>!
```

Comment:
People usually have names with spaces, but I figured it wouldn't hurt to keep things looking good here, too.